api/src/main/java/org/apache/apex/api/ApexPlugin.java

```diff
@@ -18,12 +18,15 @@
  */
 package org.apache.apex.api;
 
+import org.apache.hadoop.classification.InterfaceStability;
+
 import com.datatorrent.api.Component;
 
 /**
  * Marker interface for ApexPlugins.
  * @param <T>
  */
+@InterfaceStability.Evolving
 public interface ApexPlugin<T extends ApexPluginContext> extends Component<T>
```

Comment:
I would prefer Apex prefix and Plugin suffix to be removed as the package name already includes apex and plugin. I'd suggest "Dojo" as a common name for Dag plugins.